server/src/test/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessInstantiationResourceTest.java

```diff
@@ -118,8 +118,7 @@ public void should_instanciate_a_process_with_given_inputs() throws Exception {
         assertThat(response).hasStatus(Status.SUCCESS_OK);
         assertThat(response.getEntityAsText())
         .isEqualTo(
-                "{\"id\":\"0\",\"end_date\":\"\",\"startedBySubstitute\":\"0\",\"start\":\"\",\"state\":\"\","
-                        + "\"rootCaseId\":\"0\",\"started_by\":\"0\",\"processDefinitionId\":\"\",\"last_update_date\":\"\"}");
+                        "{\"processInstanceId\":0}");
```

Comment:
we ought to use caseId instead of processInstanceId since it doesn't mean antyhing on the web side, shouldn't we ?